src/.vscode/launch.json

```diff
@@ -8,11 +8,11 @@
             "name": "bridge",
             "type": "coreclr",
             "request": "launch",
-            //"preLaunchTask": "BuildAndPublishMac", // Replace by BuildAndPublishWindows as needed
+            "preLaunchTask": "BuildAndPublishMac", // Replace by BuildAndPublishWindows as needed
             // If you have changed target frameworks, make sure to update the program path.
             // Replace osx-x64 with win-x64 if you are running in windows
             "program": "${workspaceFolder}/dsc/bin/Debug/netcoreapp3.1/osx-x64/dsc.dll",
-            "args": ["connect", "--service", "stats-api", "--local-port", "3001", "--use-kubernetes-service-environment-variables", "--namespace", "todo-app", "--routing", "elvilla-8474"], //,
+            "args": ["connect", "--service", "stats-api", "--local-port", "3001", "--use-kubernetes-service-environment-variables", "--namespace", "todo-app"], //, "--routing", "elvilla-8474"
```

Comment:
💡 🥷 - Waht do you think? Shall we just drop this comment `//, "--routing", "elvilla-8474"` looks like personal debuggin info and we will maintain the git history anyways.

Reply:
yup, mostly just wanted to uncomment the build entry and fix build tasks. This came in by accident :)